kolibri/plugins/facility_management/assets/src/views/UserCreatePage.vue

```diff
@@ -0,0 +1,295 @@
+        ];
+      },
+    },
+    mounted() {
+      this.$store.dispatch('notLoading');
+    },
+    methods: {
+      goToUserManagementPage(onComplete) {
+        this.$router.push(this.$router.getRoute('USER_MGMT_PAGE'), onComplete);
+      },
+      usernameIsUnique(value) {
+        return !this.facilityUsers.find(
+          ({ username }) => username.toLowerCase() === value.toLowerCase()
+        );
+      },
+      submitForm() {
+        this.formSubmitted = true;
+        if (!this.formIsValid) {
+          return this.focusOnInvalidField();
+        }
+        this.busy = true;
+        this.$store
+          .dispatch('userManagement/createFacilityUser', {
+            username: this.username,
+            full_name: this.fullName,
+            id_number: this.idNumber,
+            gender: this.gender,
+            birth_year: this.birthYear,
+            role: {
+              kind: this.newUserRole,
+              collection: this.currentFacilityId,
+            },
+            password: this.password,
+          })
+          .then(() => {
+            this.handleSubmitSuccess();
+          })
+          .catch(error => {
+            this.handleSubmitFailure(error);
+          });
+      },
+      handleSubmitSuccess() {
+        this.goToUserManagementPage(() => {
+          this.$store.dispatch(
+            'createSnackbar',
+            this.$tr('userCreatedNotification', { username: this.username })
+          );
+        });
+      },
+      handleSubmitFailure(error) {
+        this.caughtErrors = CatchErrors(error, [ERROR_CONSTANTS.USERNAME_ALREADY_EXISTS]);
+        this.busy = false;
+        if (this.caughtErrors.length > 0) {
+          this.focusOnInvalidField();
+        } else {
+          this.$store.dispatch('handleApiError', error);
+        }
+      },
+      focusOnInvalidField() {
+        this.$nextTick().then(() => {
+          if (!this.fullNameValid) {
+            this.$refs.fullNameTextbox.focus();
+          } else if (!this.usernameValid) {
+            this.$refs.usernameTextbox.focus();
+          } else if (!this.passwordValid) {
+            this.$refs.passwordTextbox.focus();
+          }
+        });
+      },
+    },
+    $trs: {
+      createNewUserHeader: 'Create new user',
+      classCoachDescription: "Can only instruct classes that they're assigned to",
+      facilityCoachDescription: 'Can instruct all classes in your facility',
+      userCreatedNotification: "User account for '{username}' was created",
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .coach-selector {
+    padding: 0;
+    margin: 0;
+    border: 0;
+  }
+
+  .select {
+    margin: 18px 0 36px;
+  }
+
+  .buttons {
+    button:first-of-type {
+      margin-left: 0;
+    }
+  }
+
+  .narrow-container {
+    width: 500px;
+    margin: auto;
+  }
+
+  .form {
+    max-width: 400px;
+    min-height: 950px;
+    margin-right: auto;
+    margin-bottom: 100px;
+    margin-left: auto;
+  }
+
+</style>
```

Comment:
You may want to review the styling here, as it doesn't seem too responsive. On viewport widths less than 550px, it starts to suffer.
![screenshot-localhost-8080-2019 08 23-14-28-10](https://user-images.githubusercontent.com/819838/63624844-a1bed700-c5b2-11e9-9750-515ee5274db1.png)

Reply:
Cool. Changing the `.narrow-container` rule from `width` to `max-width` should fix this.